Source/WebCore/rendering/svg/RenderSVGResourceGradient.h

```diff
@@ -31,6 +31,33 @@ namespace WebCore {
 
 class GraphicsContext;
 
+struct GradientData {
+    WTF_MAKE_STRUCT_FAST_ALLOCATED;
+
+    struct Inputs {
+        bool operator==(const Inputs& other)
+        {
+            return objectBoundingBox == other.objectBoundingBox && textPaintingScale == other.textPaintingScale;
+        }
```

Comment:
Do we need this operator? I do not see it used. But I see we use the default operator=!() below:

`if (this->inputs != inputs) {`

Reply:
You are right, no need for this.

Reply:
Actually I am confused. Removing the `operator==` makes this line not compile.